presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -120,10 +130,13 @@ public QueryState getState()
     }
 
     @JsonProperty
-    public boolean isScheduled()
-    {
-        return scheduled;
-    }
+    public ErrorType getErrorType() { return errorType; }
```

Comment:
Per our code style, method bodies should be on separate lines (they may appear to be on the same line in IDEA because it folds them)